### DISCUSSION TITLE
Houdini Apprentice ( free version)

### DISCUSSION THREAD

**Discussion by StrainzAgent** [stassius/StableHoudini#29] [Q&A] [answered]
Just saw your videos online, mind blowing stuff! Are we able to use it with the free Houdini version ( Houdini Apprentice) ?

https://www.sidefx.com/buy/#houdini-free

**Comment by stassius** [comment, marked as answer]
Yes, it should work. The only thing is you'll get a watermark on renders made by standard render nodes like Mantra or Opengl. This could make it harder to use for rendering depth maps. But all Stable Diffusion nodes will work.
